### ISSUE TITLE
Accessibility: Object Status: Clickable object statuses are not reachable via keyboard in PC mode

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3447]
Description: Clickable object statuses are not reachable via keyboard in PC mode

Expected: Clickable/interactive UI’s should be reachable via keyboard in PC mode

screenshot:

![image](https://user-images.githubusercontent.com/32538291/94521121-20248880-024b-11eb-81ae-41af124ee498.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
